### DISCUSSION TITLE
RTMP Client

### DISCUSSION THREAD

**Discussion by amiadz-highlander** [OvenMediaLabs/OvenMediaEngine#1214] [Help]
Hi guys,

I would like to have the option to pull RTMP streams along with WEBRTC/HLS streams
I know that it is possible to push RTMP to a different server, but this will create latency and a need for a different server.
Can RTMP packets be pulled or, if not, do you intend to add support, and when?

**Comment by getroot** [comment]
Duplicate #1081
